src/main/java/net/coderbot/iris/mixin/lang/MixinTranslationStorage.java

```diff
@@ -0,0 +1,39 @@
+package net.coderbot.iris.mixin.lang;
+
+import net.coderbot.iris.Iris;
+import net.minecraft.client.resource.language.TranslationStorage;
+import net.minecraft.resource.Resource;
+import net.minecraft.util.Language;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+import org.spongepowered.asm.mixin.injection.callback.LocalCapture;
+
+import java.io.InputStream;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+@Mixin(TranslationStorage.class)
+public class MixinTranslationStorage {
```

Comment:
It would probably be better to just pull in `fabric-resource-loader` instead of doing this